tools/update-metadata.lua

```diff
@@ -65,38 +60,329 @@ local function getLocales()
+    else
+        return offset .. '<' .. element .. ' xml:lang="' .. lang .. '">' .. _(str) .. '</' .. element .. '>'
+    end
+end
+
+local metadata = {
+    summary = _("Ebook reader"),
+    desc = {
+        paragraphs = {
+            _("KOReader is an ebook reader optimized for e-ink screens. It can open many formats and provides advanced text adjustments."),
+            _("See below for a selection of its many features:"),
+        },
+        highlights = {
+            _("Supports both fixed page formats (PDF, DjVu, CBT, CBZ) and reflowable e-book formats (EPUB, FB2, Mobi, DOC, CHM, TXT, HTML). Scanned PDF/DjVu documents can be reflowed. Special flow directions for reading double column PDFs and manga."),
+            _("Multi-lingual user interface optimized for e-ink screens. Highly customizable reader view with complete typesetting options. Multi-lingual hyphenation dictionaries are bundled in."),
+            _("Non-Latin script support for books, including the Hebrew, Arabic, Persian, Russian, Chinese, Japanese and Korean languages."),
+            _("Unique Book Map and Page Browser features to navigate your book."),
+            _("Special multi-page highlight mode with many local and online export options."),
+            _("Can synchronize your reading progress across all your KOReader running devices."),
+            _("Integrated with Calibre, Wallabag, Wikipedia, Google translate and other content providers."),
+        },
+    },
+    notes = {
+        _ ("Release notes available on the link below"),
+    },
+    screenshots = {
+        { image = "https://github.com/koreader/koreader-artwork/raw/master/koreader-menu-framed.png",
+          default = true,
+        },
+        { image = "https://github.com/koreader/koreader-artwork/raw/master/koreader-footnotes-framed.png",
+        },
+        { image = "https://github.com/koreader/koreader-artwork/raw/master/koreader-dictionary-framed.png",
+        }
+    },
+    keywords = {
+        _("reader"),
+        _("viewer"),
+        _("dictionary"),
+        _("wikipedia"),
+        _("wallabag"),
+        _("annotations"),
+        -- don't translate formats
+        "epub",
+        "fb2",
+        "pdf",
+        "djvu",
+    },
+
+    -- appstream metadata don't need to be translated
+    component = [[
+  <id>rocks.koreader.KOReader</id>
+
+  <name>KOReader</name>
+  <developer id="rocks.koreader">
+    <name>KOReader Community</name>
+  </developer>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+
+  <url type="homepage">https://koreader.rocks/</url>
+  <url type="bugtracker">https://github.com/koreader/koreader/issues</url>
+  <url type="faq">https://github.com/koreader/koreader/wiki</url>
+  <url type="translate">https://hosted.weblate.org/engage/koreader/</url>
+  <url type="vcs-browser">https://github.com/koreader/koreader</url>
+  <url type="contribute">https://koreader.rocks/doc/</url>
```

Comment:
It seems that contribute should point to gh, not dev docs.

Reply:
Agreed